src/main/kotlin/dev/arbjerg/lavalink/client/loadbalancing/builtin/DefaultLoadBalancer.kt

```diff
@@ -0,0 +1,41 @@
+package dev.arbjerg.lavalink.client.loadbalancing.builtin
+
+import dev.arbjerg.lavalink.client.LavalinkClient
+import dev.arbjerg.lavalink.client.LavalinkNode
+import dev.arbjerg.lavalink.client.loadbalancing.ILoadBalancer
+import dev.arbjerg.lavalink.client.loadbalancing.VoiceRegion
+
+class DefaultLoadBalancer(private val client: LavalinkClient) : ILoadBalancer {
+    private val penaltyProviders = mutableListOf<IPenaltyProvider>()
+
+    // TODO: Why even do this LOL
```

Comment:
(in case you were wondering, this comment is referring to having the add/remove functions instead of making the list public)